docs/configuration.md

```diff
@@ -141,17 +141,33 @@ analyzers:
 
 ### Add a Custom Message to the Posted Comments
 
-You can configure **source{d} Lookout** to add a custom message to every comment that each analyzer returns. This custom message will be created following the rule:
-```
-sprintf(providers.github.comment_footer, feedback)
-```
-If any of those two keys are not defined, the custom message won't be added.
+You can configure **source{d} Lookout** to add a custom message to every comment that each analyzer returns. This custom message will be created passing to the template defined by `providers.github.comment_footer`, the `name` and `feedback` defined for each analyzer configuration.
+
+If the template (`providers.github.comment_footer`) is empty, or the analyzer configuration does not define any of the values that the template requires, the custom message won't be added.
 
 Example:
-```text
-"_If you have feedback about this comment, please, [tell us](%s)._"
+```yaml
```

Comment:
Instead of splitting the confing into two, I think it would be more clear to show it as one file. Something like:

--------

For example, for this configuration, each analyzer needs to define `name` and `feedback`:
```yaml
providers:
  github:
    comment_footer: "Comment made by analyzer {{.Name}}, [report]({{.Feedback}})."

analyzers:
  - name: Fancy Analyzer
    addr: ipv4://localhost:9930
    feedback: http://example.com/report-issue
  - name: Awesome Analyzer
    addr: ipv4://localhost:9931
```

Comments from `Fancy Analyzer` will have this footer appended:
>_Comment made by analyzer Fancy Analyzer, [report](http://example.com/report-issue)._

but comments from `Awesome Analyzer` will not have a footer message because its configuration is missing a `feedback` value.